src/textfield/textfield.ios.ts

```diff
@@ -170,8 +181,12 @@ export class TextField extends TextFieldBase {
     [floatingColorProperty.setNative](value: Color) {
         const color = value instanceof Color ? value.ios : value;
         this._controller.floatingPlaceholderActiveColor = color;
+        this._updateAttributedPlaceholder();
+    }
+    [floatingInactiveColorProperty.setNative](value: Color) {
+        const color = value instanceof Color ? value.ios : value;
+        this._controller.inlinePlaceholderColor = color;
```

Comment:
you brought `inlinePlaceholderColor` back. Any reason? Must have removed it for reason (which i don't remember right now :s )

Reply:
When testing iOS I realized I needed to set both since: 

`floatingPlaceholderNormalColor` applies when text is in the field and the field is inactive (i.e. the placeholder is floating but is not active)

`inlinePlaceholderColor` applies when no text is in the field and the field is inactive (i.e. the placeholder is not yet floating)

Both of these states apply to the inactive state of a field so I set them both

![Screen Shot 2020-02-18 at 5 19 06 PM](https://user-images.githubusercontent.com/2379994/74783105-f5189600-5272-11ea-819b-6da552570fe3.png)

Reply:
Ok i get it great.